irohad/main/server_runner.hpp

```diff
@@ -14,14 +14,18 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+#include <grpc++/grpc++.h>
 #include <grpc++/server_builder.h>
 
-namespace connection {
+#ifndef CONNECTION_SERVER_RUNNER_HPP
+#define CONNECTION_SERVER_RUNNER_HPP
 
-    class ServerRunner {
-    public:
-        ServerRunner(const std::string& ip,
-                     const std::vector<grpc::Service*>& services);
-    };
+namespace server_runner {
+  void initialize(const std::string &ip, int port,
+                  const std::vector<grpc::Service *> &services);
+  void run();
```

Comment:
> No so good make classes as "hard" implementations

Hmm... Is it better to use interface in ServerRunner class?

"My silly misunderstanding" is that `std::thread` for running server can use only static method, but it is wrong. So, I changed back the code.

By the way, currently, some modules (about not connection modules) uses functions in namespaces. I think It is because of following 2 things.
1) They are like Singleton.
2) On the whole, Iroha's data flow doesn't have states. Iroha only does receive tx, change to block, add signatures, and pass the block to others.